android-vjestina-homework-movie-app-data-mock/app/src/main/java/agency/five/codebase/android/movieapp/ui/moviedetails/MovieDetailsScreen.kt

```diff
@@ -0,0 +1,283 @@
+                        bottom = MaterialTheme.spacing.medium
+                    )
+                    .size(30.dp),
+                onClick = { }
+            )
+        }
+    }
+}
+
+@Composable
+fun MovieDetailsOverview(
+    modifier: Modifier = Modifier,
+    movieDetailsViewState: MovieDetailsViewState
+) {
+    Column(
+        modifier = modifier
+            .fillMaxWidth()
+            .padding(start = MaterialTheme.spacing.medium, end = MaterialTheme.spacing.medium)
+    ) {
+        Text(
+            text = stringResource(id = R.string.overview),
+            maxLines = 1,
+            modifier = Modifier
+                .fillMaxWidth(),
+            fontSize = 18.sp,
+            fontWeight = FontWeight.Bold
+        )
+
+        Spacer(
+            modifier = Modifier
+                .height(MaterialTheme.spacing.small)
+        )
+
+        Text(
+            text = movieDetailsViewState.overview,
+            maxLines = 7,
+            modifier = Modifier
+                .fillMaxWidth(),
+            fontSize = 14.sp
+        )
+    }
+}
+
+@Composable
+fun MovieDetailsCrew(
+    modifier: Modifier = Modifier,
+    movieDetailsViewState: MovieDetailsViewState
+) {
+    Column(
+        modifier = modifier
+            .heightIn(min = 50.dp, max = 400.dp)
+            .padding(start = MaterialTheme.spacing.medium, end = MaterialTheme.spacing.medium)
+    ) {
+        LazyVerticalGrid(
+            columns = GridCells.Fixed(3),
+            verticalArrangement = Arrangement.spacedBy(MaterialTheme.spacing.medium),
+            userScrollEnabled = false
+        ) {
+            items(
+                items = movieDetailsViewState.crew,
+                key = { crewman -> crewman.id }
+            ) { crewman ->
+                CrewItem(
+                    item = CrewItemViewState(
+                        crewman.name,
+                        crewman.job
+                    )
+                )
+            }
+        }
+    }
+}
+
+@Composable
+fun MovieDetailsCast(
+    modifier: Modifier = Modifier,
+    movieDetailsViewState: MovieDetailsViewState
+) {
+    Column(
+        modifier = modifier
+            .padding(start = MaterialTheme.spacing.medium, end = MaterialTheme.spacing.medium)
+    ) {
+        Text(
+            text = stringResource(id = R.string.top_billed_cast),
+            maxLines = 1,
+            modifier = Modifier
+                .fillMaxWidth(),
+            fontSize = 18.sp,
+            fontWeight = FontWeight.Bold
+        )
+
+        Spacer(
+            modifier = Modifier
+                .height(MaterialTheme.spacing.small)
+        )
+
+        LazyRow(
+            horizontalArrangement = Arrangement.spacedBy(MaterialTheme.spacing.medium)
+        ) {
+            items(
+                items = movieDetailsViewState.cast,
+                key = { actor -> actor.id }
+            ) { actor ->
+                ActorCard(
```

Comment:
Pass here a modifier with a fixed size for the `ActorCard`. This should fix your image loading issues.
e.g.
```kotlin
modifier = Modifier
                    .size(
                        width = xxx,
                        height = yyy
                    )
```